libcudacxx/.upstream-tests/test/std/views/views.span/span.objectrep/as_bytes.pass.cpp

```diff
@@ -39,7 +42,9 @@ void testRuntimeSpan(Span sp)
         assert(spBytes.extent == sizeof(typename Span::element_type) * sp.extent);
 
     assert((void *) spBytes.data() == (void *) sp.data());
+#ifndef TEST_COMPILER_MSVC_2017
```

Comment:
...why is this disabled on 2017?

Reply:
There is a misscompilation with MSVC2017 in exactly that place, where it fails that assert because of some integer overflow that should not happen.